.eslintrc.json

```diff
@@ -31,6 +31,12 @@
         "prefer-const": "warn",
         "use-isnan": "error",
         "unused-imports/no-unused-imports-ts": "error",
-        "lodash/import-scope": ["error", "member"]
+        "lodash/import-scope": ["error", "member"],
+        "prettier/prettier": [
+            "error",
+            {
+                "endOfLine": "auto"
```

Comment:
Hi, please don't adjust this. Did you encounter problems with CR/LF?

Reply:
plz revert the code style changes, thanks